helm-charts/konk-service/templates/configmap.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "konk-service.fullname" . }}-mounts
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "konk-service.labels" . | nindent 4 }}
+data:
+  deploy-api-service.sh: |
+    #!/bin/bash
+    set -xe
+
+    CERT=$(cat certs/ca.crt | base64)
+    CERT=${CERT//$'\n'/}
+    cat /mounts/api-service.yaml.in | sed "s/{{` {{ SERVICENAME }} `}}/${SERVICENAME}/g" | sed "s/{{` {{ NAMESPACE }} `}}/${NAMESPACE}/g" | sed "s/{{` {{ CERT }} `}}/${CERT}/g" > /gen/api-service.yaml
+    if ! kubectl get apiservice v1alpha1.$SERVICENAME.infoblox.com
+    then
+      kubectl apply -f /gen/api-service.yaml
```

Comment:
I had an error with my konk (the name didn't include `konk`) which caused it to not be created. While konk was down, this failed and never retried. We'll need try with backoff here.